src/compiler/factory.ts

```diff
@@ -2119,19 +2119,6 @@ namespace ts {
             : node;
     }
 
-    export function createBundle(sourceFiles: SourceFile[]) {

```

Comment:
Ok I will move around more functions from utilities to factory/emitter then. (This is because utilities.ts is now needed wherever you include parser.ts and that means I have to keep utilities.ts such that it doesn't need any more files than already included)